### PR TITLE
feat(python): UUID Ast Node

### PR DESCRIPTION
**Pull request by noanflaherty**
This introduces a new AstNode for creating `UUID` instances.

If this is too specific, I can keep this in our repo. I thought it was generally useful enough to include in fern though.

### CODE REVIEW

File: generators/python-v2/codegen/src/ast/Uuid.ts
Comment (noanflaherty) [reply generators/python-v2/codegen/src/ast/Uuid.ts:42]: My understanding of `Type` is that it's used to render a type hint. Such as `my_var: List[str]`. The goal of this ast node is to render an instance of a UUID, such as `my_var = UUID("<some-uuid>")`. Given this context, do you still feel that way? If so, am I thinking of `Type` incorrectly?

Comment (amckinney) [reply generators/python-v2/codegen/src/ast/Uuid.ts:42]: Got it, so in this case it would normally be a `TypeInstantiation` similar ([C#](https://github.com/fern-api/fern/blob/27e99c941c244e45e69f4f728ad50c70c410b00e/generators/csharp/codegen/src/ast/InstantiatedPrimitive.ts#L134) and [Go](https://github.com/fern-api/fern/blob/27e99c941c244e45e69f4f728ad50c70c410b00e/generators/go-v2/codegen/src/ast/TypeInstantiation.ts#L172)).

It's not the end of the world to include this here as-is, but I'd be wary of creating separate AST nodes for other primitive type instantiations like this since mainly because it's inconsistent.

**Review by amckinney — APPROVED** [review]
Stamping to unblock - I'll leave it to you to decide what makes sense in this context (especially because it's fairly easy to change later if need be).

File: generators/python-v2/codegen/src/ast/Uuid.ts
Comment (noanflaherty) [reply generators/python-v2/codegen/src/ast/Uuid.ts:42]: Ah okay, I'll close this PR and introduce a separate `TypeInstantiation` AstNode.

**Pull request closed without merging**
